app/javascript/components/public_health/PublicHealthHeader.js

```diff
@@ -90,14 +90,11 @@ class PublicHealthHeader extends React.Component {
         <Modal.Header closeButton>
           {this.state.importData && (
             <React.Fragment>
-              {this.state.importData.errors.length > 0 && (
-                <Modal.Title className="h5">{this.state.importType === 'epix' ? 'Import Epi-X' : 'Import Sara Alert Format'} (error)</Modal.Title>
-              )}
-              {this.state.importData.errors.length === 0 && (
-                <Modal.Title className="h5">
-                  {this.state.importType === 'epix' ? 'Import Epi-X' : 'Import Sara Alert Format'} ({this.props.workflow})
-                </Modal.Title>
-              )}
+              <Modal.Title className="h5">
+                {this.state.importType === 'saf' && 'Import Sara Alert Format'}
+                {this.state.importType === 'epix' && 'Import Epi-X'}
+                {this.state.importType === 'sdx' && 'Import SDX'} {this.state.importData.errors.length === 0 ? this.props.workflow : '(error)'}
```

Comment:
I had @deastman make this change on her PR so for consistency, are you against changing `(error)` to `Error`?

Reply:
See [here](https://github.com/SaraAlert/SaraAlert/pull/1168#pullrequestreview-828926764) for discussion

Reply:
https://github.com/SaraAlert/SaraAlert/pull/1170/commits/0608463add7b71268c7debfd216a8b650d8d02af